package.json

```
@@ -4,7 +4,7 @@
   "description": "The Bitcrowd CSS library",
   "main": "build/bitstyles.css",
   "scripts": {
-    "build": "sass scss/bitstyles.scss build/bitstyles.css --style=compressed --no-source-map",
+    "build": "sscss/bitstyles.scss build/bitstyles.css --style=compressed --no-source-map",
```

This line looks like it got mangled somehow 🙈
```suggestion
    "build": "sass scss/bitstyles.scss build/bitstyles.css --style=compressed --no-source-map",
```